README.md

```diff
@@ -1,10 +1,65 @@
-# hdbpp-cm
+# Hdbpp-cm
+
 Tango device server for the HDB++ Configuration Manager
 
-## building
-git clone --recursive http://github.com/tango-controls/hdbpp-cm.git  
-cd hdbpp-cm  
-export TANGO_DIR=/usr/local/tango-9.2.5a  
-export OMNIORB_DIR=/usr/local/omniorb-4.2.1  
-export ZMQ_DIR=/usr/local/zeromq-4.0.7  
+## Version
+
+The current release version is 1.0.0
+
+## Documentation
+
+* See the Tango documentation [here](http://tango-controls.readthedocs.io/en/latest/administration/services/hdbpp/index.html#hdb-an-archiving-historian-service) for broader information about the HB++ archiving system and its integration into Tango Controls
+* hdbpp-cm [CHANGELOG.md](https://github.com/tango-controls-hdbpp/libhdbpp/blob/master/CHANGELOG.md) contains the latest changes both released and in development.
+
+## Building
+
+### Dependencies
+
+Ensure the development version of the dependencies are installed. These are as follows:
+
+* HDB++ library libhdbpp
+* Tango Controls 9.2.5a - either via debian package or source install.
+* omniORB release 4 - libomniorb4 and libomnithread
+* libzmq - libzmq3-dev or libzmq5-dev
+
+### Build Flags
+
+There are a set of library and include variables that can be set to inform the build of various dependencies. The flags are only required if you have installed a dependency in a non-standard location. 
```

Comment:
set of library => set of libraries

Reply:
Actually, the plural is on the variables - "library and include variables" as in, library variables and include variables.

Reply:
Ah!? OK! Makes sense ;-)